tandem_aligner/src/projects/tandem_aligner/min_interval.hpp

```diff
@@ -80,11 +81,14 @@ class MinIntervalFinder {
     bool force_highfreq_search{false};
     bool exprt{true};
     std::experimental::filesystem::path outdir;
+    bool max_unique{false};
+    logging::Logger &logger;
 
     struct MinRarePrefix {
```

Comment:
Please make sure to rename Min to MinMax where appropriate since you are saving both